src/main/resources/react/src/pages/UserInfo/tabs/tabs.js

```diff
@@ -1,43 +1,45 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
+import { useSelector } from 'react-redux';
 
 import { route } from '../../../routes/routeConstants';
+import { getTranslatedText } from '../../../components/local/localisation';
 
 import "./tabs.scss";
 
 
 const Tabs = props => {
   const { url } = props;
-
+  const { lang } = useSelector( state => state.auth );
   return (
     <div className="tabs">
       <NavLink to={`${url}`} exact >
         <i className={"icon-activity"} />
-        Моя активность
+        {getTranslatedText('ownInfo.panelMyActivity',lang)}
         <i className="active__cycle" />
       </NavLink>
```

Comment:
I see duplication of the same logic 4 times here. Could it be refactored kinda that:

```
function createNavLink(url, iconClassName, textKey, lang) {
    return <NavLink to={ url } >
        <i className={ iconClassName } />
        {getTranslatedText( textKey, lang)}
        <i className="active__cycle" />
      </NavLink>
}
```